guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
 
 # Funding a wallet
 
-Common problems faced by new users include buying their first bitcoin and transferring it to a new wallet. For this, most will purchase their first bitcoin from an exchange using a national or fiat currency. This generally involves providing some form of documentation (such as ID) and already having a bank account. After purchasing their bitcoin, they can then send it to an external wallet. While this works, there are better alternatives available. Understanding how your users fund their wallets can help you create more tailored and dedicated UI flows. Certain ways users fund their wallets are outlined below.
+A user will need to fund their wallet with some bitcoin to begin using it. When the user funds a wallet for the first time they are likely to be introduced to new concepts that need to be clearly explained. The design flows around funding a wallet can range from simple to complex, depending on your application's target audience and which services you offer to streamline the process.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
+   layout = "shadow"
+   caption = "Ask the user if they own bitcoin."
```

Comment:
Do you need the two dots? I assume the paths are different based on what choice you make, so you might not know how many steps are still to come.